test/integration/test_integration.py

```diff
@@ -2076,6 +2078,26 @@ def forward(self, x):
             self.assertTrue(torch.ops.torchao.choose_qparams_affine.default in targets)
             self.assertTrue(torch.ops.torchao.quantize_affine.default in targets)
             self.assertFalse(torch.ops.aten.narrow.default in targets)
+    
+    def test_export_float8(self):
```

Comment:
need this skip according to CI: https://github.com/pytorch/ao/blob/c2a6568a04075acc371a338206216bb65536fb27/test/dtypes/test_affine_quantized_float.py#L74-L76